Sources/Items/UniversalCollectionViewCellItem.swift

```diff
@@ -38,4 +38,8 @@ public final class UniversalCollectionViewCellItem<T: UICollectionViewCell>: Col
     public func size(in collectionView: UICollectionView, sectionItem: CollectionViewSectionItem) -> CGSize {
         return sizeConfigurationHandler?(collectionView, sectionItem) ?? .zero
     }
+
+    public func isEqual(to item: DiffItem) -> Bool {
+        diffIdentifier == item.diffIdentifier
```

Comment:
You don't need to compare diffIdetifiers in `isEqual` method because its already compare in diff algorithms and `isEqual` method will called only for items with same identifiers so its always be true
```suggestion
      true
```